tutorial/creating_packages/create_your_first_package.rst

```diff
@@ -318,7 +338,6 @@ An **important** note: the Conan cache is private to the Conan client - modifyin
 Read more
 ---------
 
-- :ref:`Conan list command reference<reference_commands_list>`.
-- :ref:`Create your first Conan package with Autotools<examples_tools_autotools_autotools_toolchain_build_project_autotools_toolchain>`.
-- :ref:`Create your first Conan package with Meson<examples_tools_meson_toolchain_build_simple_meson_project>`.
-- :ref:`Create your first Conan package with CMake<examples-tools-cmake-toolchain-build-project-presets>`.
```

Comment:
These were confusing, they are pointing to "consumer" projects, not to package creation.